ui/components/ui/sender-to-recipient/index.scss

```diff
@@ -27,7 +27,7 @@
         white-space: nowrap;
         overflow: hidden;
         text-overflow: ellipsis;
-        border: 1px solid var(--color-border-default);
+        border-bottom: 1px solid var(--color-border-muted);
 
         &--sender {
           padding-right: 30px;
```

Comment:
Making border not so contrasting

### Before
<img width="719" alt="Screen Shot 2022-05-16 at 4 44 23 PM" src="https://user-images.githubusercontent.com/8112138/168916821-430cf16b-c937-4892-aeba-26ad9fffec7b.png">

### After
![Screen Shot 2022-05-17 at 2 58 16 PM](https://user-images.githubusercontent.com/8112138/168916913-592101d0-a925-46c7-8a5e-3bf80698e698.png)